contrib/bash-completion

```diff
@@ -23,9 +23,17 @@ _netctl() {
         COMPREPLY=( $(compgen -W "--help --version list store restore stop-all start stop restart switch-to is-active status enable disable reenable is-enabled edit wait-online" -- "$cur") )
       ;;
       2)
-        [[ ${COMP_WORDS[COMP_CWORD-1]} = @(start|stop|restart|switch-to|is-active|status|enable|disable|reenable|is-enabled|edit|wait-online) ]] &&
+        [[ ${COMP_WORDS[COMP_CWORD-1]} = @(is-active|status|enable|disable|reenable|is-enabled|edit|wait-online) ]] &&
           compopt -o filenames &&
           mapfile -t COMPREPLY < <(IFS=$'\n'; compgen -W "$(_netctl_profiles)" -- "$cur")
+        [[ ${COMP_WORDS[COMP_CWORD-1]} = @(stop|restart) ]] &&
+          compopt -o filenames &&
+	  # Suggest only active profiles for stopping/restarting
+          mapfile -t COMPREPLY < <(IFS=$'\n'; compgen -W "$(netctl list | grep '\*' | cut -f2 -d' ')" -- "$cur")
```

Comment:
Instead of `grep '\*' | cut -f2 -d' '`, what do you think of `sed -n 's/^[^ ] //p'`? This has the added benefit that profiles that are currently being started are included too.

Reply:
`grep` vs `sed` - I don't care as long as it gets the job done :)  
Sorry, forgot about "+" profiles. I think it is better to have a positive match rather than a negative - if you later introduce some other profile status signs like "-" they will get caught here automatically. With `sed` that would be:

```suggestion
          mapfile -t COMPREPLY < <(IFS=$'\n'; compgen -W "$(netctl list | sed -n 's/^[*+] //p')" -- "$cur")
```

Reply:
This is not forward compatible, as it has to be kept in sync with the output of `netctl list`. My suggestion is more robust.

Reply:
But what if you will introduce e. g. "-" profile status that will not be suitable for stop/switch-to suggestion? `sed -n 's/^[^ ] //p'` will catch it automatically, `sed -n 's/^[*+] //p'` won't, isn't it better to be explicit here?